src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
Yeah, but I think this way its easier to get them, as theres only 4 options and `discrim % 5` will always yield between 0 and 4, and its probably easier